bin/git-ffwd

```diff
@@ -34,12 +34,11 @@ main() {
   REMOTES=$(echo "$REMOTES" | xargs -n1 echo)
   CLB=$(git rev-parse --abbrev-ref HEAD);
   echo "$REMOTES" | while read REMOTE; do
+    git remote update $REMOTE
```

Comment:
Pls note that I’ve intentionally removed the update here. As I’ve created additional aliases like “git ruffwd” which also does the remote update. 

https://github.com/muhqu/dotfiles/blob/master/.gitconfig#L18

Reply:
ok